Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM kpndigital/tox:latest
+FROM kpndigital/tox:py27_py35
```

Comment:
Hmm the 'latest' tox doesn't support python3.6 but it has been added to this specific version. There was a long discussion on this but python36 has been added to this to prevent breaking backwards compatibility